bootcamp/tutorials/quickstart/hybrid_demo.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install --upgrade pymilvus "pymilvus[model]" milvus-lite pandas numpy
</pre><p>remove milvus-lite, pandas, numpy since they are already required by pymilvus</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>